src/style.css

```diff
@@ -0,0 +1,140 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.flexbox {
+  display: flex;
+  flex-direction: column;
+}
+
+.box {
+  display: flex;
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  background-color: red;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.box-2 {
+  order: 1;
+  background-color: rgb(204, 0, 0);
+}
+
+.box-3 {
+  order: 2;
+  background-color: rgb(153, 0, 0);
+}
+
+.box-4 {
+  order: 3;
+  background-color: rgb(102, 0, 0);
+}
+
+.box-5 {
+  order: 4;
+  background-color: rgb(51, 0, 0);
+}
+
+.box-6 {
+  order: 5;
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 450px) {
+  .box {
+    flex-basis: 300px;
+  }
+}
+
+@media (min-width: 600px) {
+  .flexbox {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .box {
+    flex-basis: 300px;
+  }
+  .box-3 {
+    order: 3;
+  }
+  .box-4 {
+    order: 2;
+  }
+}
+
+@media (min-width: 750px) {
+  .flexbox {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .box {
+    flex-basis: 375px;
+  }
+  .box-3 {
+    order: 3;
+  }
+  .box-4 {
+    order: 2;
+  }
+}
+@media (min-width: 900px) {
+  .flexbox {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .box {
+    flex-basis: 300px;
+  }
+  .box-4 {
+    order: 5;
+  }
+  .box-5 {
+    order: 4;
+  }
+  .box-6 {
+    order: 3;
+  }
+}
+@media (min-width: 1050px) {
+  .flexbox {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .box {
+    flex-basis: 350px;
+  }
+  .box-4 {
+    order: 5;
+  }
+  .box-5 {
+    order: 4;
+  }
+  .box-6 {
+    order: 3;
+  }
+}
+@media (min-width: 1200px) {
+  .flexbox {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .box {
+    flex-basis: 33%;
+  }
+  .box-4 {
+    order: 5;
+  }
+  .box-5 {
+    order: 4;
+  }
+  .box-6 {
+    order: 3;
+  }
+}
```

Comment:
Omg, you don't need such a count of media queries in this task. Try to improve this, using only 2 media queries🙂